Content.Server/Body/Systems/SkeletonBodyManagerSystem.cs

```diff
@@ -0,0 +1,187 @@
+using Content.Server.Body.Components;
+using Content.Server.Cloning;
+using Content.Server.Construction;
+using Content.Server.DoAfter;
+using Content.Server.Mind.Components;
+using Content.Server.Popups;
+using Content.Server.Preferences.Managers;
+using Content.Shared.CharacterAppearance.Systems;
+using Content.Shared.Preferences;
+using Content.Shared.Species;
+using Content.Shared.Verbs;
+using Robust.Server.GameObjects;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using System.Diagnostics.CodeAnalysis;
+
+/// <remarks>
+/// Fair warning, this is all kinda shitcode, but it'll have to wait for a major
+/// refactor until proper body systems get added. The current implementation is
+/// definitely not ideal and probably will be prone to weird bugs.
+/// </remarks>
+
+namespace Content.Server.Body.Systems
+{
+    public sealed class SkeletonBodyManagerSystem : EntitySystem
+    {
+        [Dependency] private readonly IEntityManager _entities = default!;
```

Comment:
all entity systems have an innate reference to entity manager called `EntityManager`, you dont need this